src/prebid.js

```diff
@@ -847,4 +847,12 @@ $$PREBID_GLOBAL$$.getHighestCpmBids = function (adUnitCode) {
   return getWinningBids(adUnitCode);
 };
 
+/**
+ * Returns the adaptermanager.bidderRegistry
+ * @returns {*}
+ */
+$$PREBID_GLOBAL$$.getBidderRegistry = function() {
```

Comment:
Why is this added to the global? We shouldn't need to expose this. If we need a means to store objects let's put it into a separate file that does not have global access.